src/main/java/org/minions/devfund/chelo/Squarelotron.java

```diff
@@ -0,0 +1,185 @@
+package org.minions.devfund.chelo.squarelotron;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "Squarelotron.java" should be located in "org/minions/devfund/chelo/squarelotron" directory, not in "/home/travis/build/Minions-F/devfund/src/main/java/org/minions/devfund/chelo". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)